doc/internals/contributing.rst

```diff
@@ -138,10 +138,7 @@ Coding style
 
 Please follow these guidelines when writing code for Sphinx:
 
-* Try to use the same code style as used in the rest of the project.  See the
-  `Pocoo Styleguide`__ for more information.
-
-  __ http://flask.pocoo.org/docs/styleguide/
+* Try to use the same code style as used in the rest of the project.
```

Comment:
I think it would be better to update to the new styleguide rather than maintain the old one here